lib/txgh/transifex_api.rb

```diff
@@ -50,9 +50,15 @@ def create_or_update(tx_resource, content, categories = [])
     end
 
     def create(tx_resource, content, categories = [])
+      name = if tx_resource.branch
+        "#{tx_resource.source_file} (#{tx_resource.branch})"
```

Comment:
Yeah, it's not restricted in any way I know of. I have manually added branch names to these guys before and Transifex doesn't seem to care if there are spaces.